src/main/java/frc/trigon/robot/subsystems/gripper/GripperConstants.java

```diff
@@ -0,0 +1,93 @@
+package frc.trigon.robot.subsystems.gripper;
+
+import au.grapplerobotics.ConfigurationFailedException;
+import au.grapplerobotics.LaserCan;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.math.system.plant.DCMotor;
+import org.trigon.hardware.grapple.lasercan.LaserCAN;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXMotor;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXSignal;
+import org.trigon.hardware.simulation.SimpleMotorSimulation;
+import org.trigon.utilities.mechanisms.SpeedMechanism2d;
+
+import java.util.function.DoubleSupplier;
+
+
+public class GripperConstants {
+    private static final int
+            MOTOR_ID = 15,
+            LASER_CAN_ID = 5;
+
+    static final TalonFXMotor MOTOR = new TalonFXMotor(MOTOR_ID, "GripperMotor");
+    static final LaserCAN LASER_CAN = new LaserCAN(LASER_CAN_ID, "GripperLaserCAN");
+
+    private static final InvertedValue INVERTED_VALUE = InvertedValue.CounterClockwise_Positive;
+    private static final NeutralModeValue NEUTRAL_MODE_VALUE = NeutralModeValue.Brake;
+    private static final LaserCan.RangingMode LASER_CAN_RANGING_MODE = LaserCan.RangingMode.SHORT;
+    private static final LaserCan.TimingBudget LASER_CAN_LOOP_TIME = LaserCan.TimingBudget.TIMING_BUDGET_33MS;
+    private static final DoubleSupplier LASER_CAN_SIMULATION_SUPPLIER = LASER_CAN::getDistanceMillimeters;
+    static final boolean FOC_ENABLED = true;
+
+    private static final int
+            X = 8,
+            Y = 8,
+            W = 16,
+            H = 16;
+
+    private static final double
+            GEAR_RATIO = 0.5,
+            MOMENT_OF_INERTIA = 0.003,
+            MAXIMUM_DISPLAYABLE_VELOCITY = 12;
+
+    private static final int NUMBER_OF_MOTORS = 1;
+    private static final DCMotor GEARBOX = DCMotor.getKrakenX60(NUMBER_OF_MOTORS);
+    static final SpeedMechanism2d GRIPPER_MECHANISM = new SpeedMechanism2d("GripperMechanism", MAXIMUM_DISPLAYABLE_VELOCITY);
+    private static final SimpleMotorSimulation GRIPPER_SIMULATION = new SimpleMotorSimulation(GEARBOX, GEAR_RATIO, MOMENT_OF_INERTIA);
+
+    static {
+        configureMotor();
+        try {
+            configureLaser();
+        } catch (ConfigurationFailedException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
1. Don't throw a runtime exception, that will crash the code. 
2. Move the try catch to the relevant part in configureLaserCAN
3. If it fails just print stack trace